src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -5,3 +5,275 @@ title: NUbook
+2. Create a new branch for your changes, in the format `your_surname/short_description_of_change`:
+
+   ```sh
+   git checkout -b paye/add_2019_debrief
+   ```
+
+3. Make your changes by adding or editing MDX files in [`src/book/`](src/book/). See below for how to write and organise pages.
+4. Preview your changes by visiting [localhost:8000](http://localhost:8000) in a browser.
+5. Commit and push your changes to GitHub.
+6. Go to the repo on GitHub and create a pull request for your branch. Your pull request will be reviewed, merged, and deployed to the live site.
+
+More information on using Git can be found on the [Git Guide](/guides/general/git).
+
+NUbook content should be accessible and inclusive. A guide on making content accessible and inclusive can be found on the [Australian Government website](https://guides.service.gov.au/content-guide/accessibility-inclusivity/).
+
+## Writing Markdown
+
+NUbook content is written using [MDX](https://mdxjs.com/), an extension of [Markdown](https://daringfireball.net/projects/markdown/) with support for dynamic content via [JSX](https://reactjs.org/docs/introducing-jsx.html) and [React](https://reactjs.org/) components. Markdown provides a minimal syntax for writing and styling text content.
+
+GitHub has [a good guide](https://guides.github.com/features/mastering-markdown/) for getting started with Markdown. There's also [a short video series](https://www.youtube.com/playlist?list=PLu8EoSxDXHP7v7K5nZSMo9XWidbJ_Bns3) on using Markdown.
+
+NUbook has a [kitchen sink](/kitchen-sink) demonstrating the various functionalities avaliable to use when writing NUbook pages.
+
+### Images
+
+To add images to a NUbook page
+
+1. Add the image file in an `images` folder in the same directory as the file being edited
+2. Reference the image in Markdown:
+
+```md
+![Image description for search engines and screen readers](./images/image.png 'Image caption')
+```
+
+The image caption will be displayed below the image and is an optional component.
+
+The image description should present the content and function of the image. Find guides on writing good alternative text at [WebAIM](https://webaim.org/techniques/alttext/).
+
+### Syntax-Highlighted Code Blocks
+
+Use triple backticks ` ``` ` on separate lines to open and close code blocks. Specify the language with a file extension after the opening backticks for syntax highlighting.
+
+Example C++ code block:
+
+````md
+```cpp
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!";
+    return 0;
+}
+```
+````
+
+### Maths Symbols and Equations
+
+- For inline math, wrap TeX-formatted content with a `$`:
+
+  ```md
+  The equation is $c^2 = a^2 + b^2$.
+  ```
+
+- For math blocks, wrap with `$$` on separate lines:
+
+  ```md
+  $$
+  e^{i\phi} = \cos(\phi) + i\sin(\phi)
+  $$
+  ```
+
+NUbook uses [KaTeX](https://katex.org/) to render math. See https://katex.org/docs/support_table.html for supported symbols and functions.
```

Comment:
This is the only link which doesn't have a name/different clickable text on this page

Reply:
Can be changed to "See the [KaTeX support table](#) for ...", but I think it's fine either way. There's no confusion what it is and the current link is short and descriptive.

Reply:
fair enough